app/AutoAssign.php

```diff
@@ -346,6 +346,11 @@ public function getAvailableUsersQuery(): QueryGenerator
 			->addCondition('status', 'Active', 'e')
 			->addCondition('available', 1, 'e')
 			->addCondition('auto_assign', 1, 'e');
+		if ($this->get('working_hours')) {
+			$currentTime = date('H:i');
+			$queryGenerator->addNativeCondition(['<=', 'start_hour', $currentTime])
+				->addNativeCondition(['>', 'end_hour', $currentTime]);
```

Comment:
why u use addNativeCondition instead of addCondition?

Reply:
Start and end hours are a picklist fields, not a time